widgets.js

```diff
@@ -2550,12 +2552,19 @@ DialogBoxMorph.prototype.fixLayout = function () {
                 this.padding,
                 th + this.padding
             )));
-            this.silentSetWidth(this.body.width() + this.padding * 2);
+            this.silentSetWidth(Math.max(
+                this.body.width() + this.padding * 2,
+                this.minWidth
+            ));
             this.silentSetHeight(
                 this.body.height()
                     + this.padding * 2
                     + th
             );
+            this.body.setLeft(
```

Comment:
Why is this changed for the generic DialogBoxMorph?

Reply:
This is necessary to center the body while supporting min-width attribute for dialog boxes c34be51b99209ab97eb7eb58c2d3b98bf8860b6c
Maybe I should pull these out into their own PR and merge that in first.